source/MaterialXGraphEditor/Graph.cpp

```diff
@@ -2753,6 +2753,10 @@ void Graph::deleteLinkInfo(int startAttr, int endAttr)
     int upNode = getNodeId(startAttr);
     int downNode = getNodeId(endAttr);
 
+    if (upNode == -1 || downNode == -1) {
+        return ;
+    }
+
```

Comment:
Should we keep this? Or just remove it and let it crash when bad things happen

Reply:
I think we should keep this in order to avoid crashing when possible.

Reply:
Agreed! Should I add something like `std::err << ` to log the error instead of just return?

Reply:
Since this is an unexpected case, and the additional information would only be valuable to developers, I think it's fine to leave this as a silent handling of the error.